source/code-snippets/usage-examples/changeStream.js

```diff
@@ -31,7 +31,9 @@ async function run() {
         });
         // wait to close `changeStream` after the listener receives the event
         setTimeout(async () => {
-          resolve(await changeStream.close());
+          resolve(
```

Comment:
We recommend rewriting this example in its entirety to make it less confusing to users and mimic a more natural way that this may be implemented; perhaps a new ticket could be opened for the effort

Reply:
Sounds good, I have [made a ticket](https://jira.mongodb.org/browse/DOCSP-18158) to capture this unit of work for the DevEd DBX team.